site/pages/dashboard.test.tsx

```diff
@@ -0,0 +1,229 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import renderer from "react-test-renderer";
+import { describe, it, expect, vi, afterEach } from "vitest";
+import Dashboard, { DashboardDisruption, getServerSideProps } from "./dashboard.page";
+import * as dynamo from "../data/dynamo";
+import { databaseData } from "../testData/mockData";
+
+const getDisruptionsSpy = vi.spyOn(dynamo, "getDisruptionsDataFromDynamo");
```

Comment:
Might be worth adding `vi.mock("../data/dynamo");` in above this, just so if we had a test which didn't mock the returned value, it wouldn't try to actually connect to dynamo